### PR TITLE
PDR-26: Search functionality implemented.

### PR DESCRIPTION
**Pull request by meredithom**
TODO:
-Filtering (will create another PR to keep API changes separate) 
-Feedback/button functionality (covered by https://github.com/bcgov/parks-data-register/issues/27)

### CODE REVIEW

File: pdr-admin/src/app/services/search.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  async fetchData(text) {
+    let queryParams = {};
+    if (text) {
+      queryParams['text'] = text;
+    }
+
+    // Calling API with status = null gives you current and historical
+    let res = await this.apiService.get('search', queryParams);
+    let data = this.apiService.getArrayFromSearchResults(res);
```
Comment (marklise) [inline pdr-admin/src/app/services/search.service.ts:20]: both `const`

File: pdr-admin/src/env.js [hunk 2]
```diff
@@ -9,7 +9,7 @@
   // Environment name
   window.__env.ENVIRONMENT = "local"; // local | dev | test | prod
 
-  window.__env.API_LOCATION = "http://localhost:3000";
+  window.__env.API_LOCATION = "https://zloys5cfvf.execute-api.ca-central-1.amazonaws.com";//"http://localhost:3000";
```
Comment (marklise) [inline pdr-admin/src/env.js:12]: I don't think we should change this.

**Review by marklise — APPROVED** [review]
/w nits.

File: pdr-admin/src/env.js (hunk 2 above)
Comment (meredithom) [reply pdr-admin/src/env.js]: fixed in [c37476f](https://github.com/bcgov/parks-data-register/pull/192/commits/c37476fa5ca9a4dc9435b661c605838b66b9b871)

File: pdr-admin/src/app/services/search.service.ts (hunk 1 above)
Comment (meredithom) [reply pdr-admin/src/app/services/search.service.ts]: fixed in [c37476f](https://github.com/bcgov/parks-data-register/pull/192/commits/c37476fa5ca9a4dc9435b661c605838b66b9b871)

**Review by danieltruong — APPROVED** [review]

**Pull request merged**